src/app/app-routing.module.ts

```diff
@@ -16,7 +17,8 @@ import { ProfileComponent } from './profile/profile.component';
         { path: '', component: AlbumsComponent },
         { path: 'album/:id', component: AlbumComponent },
         { path: 'search/:query', component: SearchResultComponent },
-        { path: 'profile', component: ProfileComponent, canActivate: [ RolesGuard ], data: { role: "settingsRole" } }
+        { path: 'profile', component: ProfileComponent, canActivate: [ RolesGuard ], data: { role: 'settingsRole' } },
+        { path: 'settings/user', component: UserSettingsComponent }
```

Comment:
This will become a mess once we have implemented the half of the features airsonic will ship.
We need to split this in child routes and child component.

Also not having a 'settings' root component is bad since we don't have the ability to move between the different settings section.

Be sure to add the role required to use the component (if user is not admin he should not be able to access the component).